.github/workflows/linux.yml

```diff
@@ -0,0 +1,25 @@
+name: Linux desktop CI
```

Comment:
We may indicate that it is a workflow for `desktop kDrive` (this would help to narrow search in file finders).

Reply:
You are right. I renamed it in latest commit